packages/common/styles/_mixins.scss

```diff
@@ -0,0 +1,49 @@
+@import './tokens';
+
+// Font fallbacks
+// Do not use outside this file - these are temporary.
+// TODO: Remove these fallbacks and their usages once fonts
+// are being served remotely:
+// https://codedotorg.atlassian.net/browse/DSCO-7
+$dsco-fallback-normal: $dsco-font-family, 'Gotham 4r', sans-serif;
+$dsco-fallback-normal-italic: $dsco-font-family, 'Gotham 4i', sans-serif;
+$dsco-fallback-bold: $dsco-font-family, 'Gotham 5r', sans-serif;
+$dsco-fallback-bold-italic: $dsco-font-family, 'Gotham 5i', sans-serif;
+$dsco-fallback-bolder: $dsco-font-family, 'Gotham 7r', sans-serif;
+$dsco-fallback-bolder-italic: $dsco-font-family, 'Gotham 7i', sans-serif;
+
+@mixin font-normal {
```

Comment:
Curious - is there a reason we're naming this just `font-normal` vs something more specific to its font face or function (like `gotham-normal` or `paragraph-font-normal`, something like that)? Are we only planning on having one font face or multiple for different functions/styles?

Reply:
hmmm, yeah -- i named these generically so that it would be easy to change the underlying font that this mixin returns, but maybe that's less flexible because it doesn't leave room for other font mixins. we've only ever used Gotham, but that doesn't mean we should plan on that always being the case